### PR TITLE
Update wording for stale bot

### PR DESCRIPTION
**Pull request by Pandapip1**
~~Blocked by #5365~~

### CODE REVIEW

**Comment by Pandapip1** [comment]
CC @MicahZoltu

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -20,14 +20,14 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         ascending: true # Since we have so many issues, the stale bot finds it hard to keep track. This makes sure that at least the oldest are removed.
         # Issue config
-        stale-issue-message: There has been no activity on this issue for six months. It will be closed in a week if no further activity occurs. If you would like to move this EIP forward, please respond to any outstanding feedback or add a comment indicating that you have addressed all required feedback and are ready for a review.
+        stale-issue-message: There has been no activity on this issue in six months. It will be closed in two weeks if no further activity occurs. If you are still pursuing it, feel free to respond to any feedback.
```
Comment (MicahZoltu) [inline .github/workflows/stale.yml:23]: For the record, I believe `for` is just as correct as `in` [in|for] this case.  I don't really care which we use, but I'm curious why the desire to change it?

Regarding the rest of the change, I feel like the original wording gave authors more clear instructions on how to proceed.  What is the perceived advantage of using the more concise statement?

Comment (Pandapip1) [reply .github/workflows/stale.yml:23]: > For the record, I believe for is just as correct as in [in|for] this case. I don't really care which we use, but I'm curious why the desire to change it?

In the stale bot documentation, the phrasing I used above is roughly what is used (including the in).

Comment (MicahZoltu) [reply .github/workflows/stale.yml:23]: Is there a more complete argument for using the proposed wording over our current wording besides "it is how someone else on the internet does it"?

Comment (Pandapip1) [reply .github/workflows/stale.yml:23]: Uh, no, but "It is how most other people on the internet do it" works better.

Comment (MicahZoltu) [reply .github/workflows/stale.yml:23]: Most humans are wrong about most things, so I don't find that to be a particularly compelling argument.  😆

Comment (Pandapip1) [reply .github/workflows/stale.yml:23]: > Most humans are wrong about most things, so I don't find that to be a particularly compelling argument. 😆

[citation needed]

**Comment by Pandapip1** [comment]
@MicahZoltu any objections?

File: .github/workflows/stale.yml (hunk 1 above)
Comment (Pandapip1) [reply .github/workflows/stale.yml:23]: I think most people would disagree with you, FYI.

Comment (Pandapip1) [reply .github/workflows/stale.yml:23]: > Regarding the rest of the change, I feel like the original wording gave authors more clear instructions on how to proceed. What is the perceived advantage of using the more concise statement?

Somehow I missed this. I think that in either scenario, it's clear that there is an automated system that is noting that the EIP has a lack of activity, and that the way to resolve this is by creating some activity. The advantage of being more concise is that it's more concise.

**Comment by MicahZoltu** [comment]
My position is still the same: I think the old wording was more clear and gave users better directions on how to proceed compared to the new wording.  Given that we disagree on which is preferred, I would like to get more editor feedback before moving forward with this change.

**Pull request closed without merging**